app/views/point_of_interests/form_partials/_lunch_offers_form.html.erb

```diff
@@ -8,11 +8,9 @@
         <div class="card mb-4">
           <div class="d-flex flex-wrap justify-content-between card-header py-3 bg-dark text-white">
             <h4>Gericht</h4>
-            <input
-              type="button"
-              value="dieses Gericht entfernen"
-              class="remove btn btn-sm btn-danger"
-            />
+            <%= link_to "/",  class: "remove btn btn-sm btn-danger" do %>
+              <i class="fa fa-trash text-white"></i>
```

Comment:
in tabellen ist das icon schöner, hier irgendwie mit zu viel padding nach unten im roten bereich

|index|edit|
|---|---|
|![Bildschirmfoto 2021-07-08 um 17 49 34](https://user-images.githubusercontent.com/1942953/124952873-e1f79900-e014-11eb-8086-ae36bddc72e5.png)|![Bildschirmfoto 2021-07-08 um 17 49 42](https://user-images.githubusercontent.com/1942953/124952883-e459f300-e014-11eb-88a0-7b259ca8526c.png)|

Reply:
grund is ein margin bottom der <h4 in der gleichen row.
ohne siehts so aus

![Bildschirmfoto 2021-07-08 um 17 50 47](https://user-images.githubusercontent.com/1942953/124953019-0bb0c000-e015-11eb-9ff2-84d0965e298f.png)

Reply:
done in 74923a2cef87fde22e79a654311cf154d9b6f334